src/programs/constrained_search/utils/get_centre_vector.py

```diff
@@ -0,0 +1,228 @@
+"""Calculates the centre vector between two PDB structures."""
+
+import argparse
+import math
```

Comment:
Use `np.pi` over `math.pi` to reduce the number of imports. I don't see the math module being used for anything but the constant